requirements.txt

```diff
@@ -1,4 +1,6 @@
+-f https://download.pytorch.org/whl/nightly/cu118
 torch>=2.0.1
 safetensors==0.3.1
 sentencepiece>=0.1.97
 ninja==1.11.1
+torch>=2.0.1
```

Comment:
This is redundant, torch is already defined

Reply:
Thanks, removed it.